aidants_connect_web/views/FC_as_FS.py

```diff
@@ -0,0 +1,104 @@
+import logging
+import requests as python_request
+from secrets import token_urlsafe
+
+from django.http import HttpResponseForbidden
+from django.shortcuts import redirect
+from django.conf import settings
+from django.utils import timezone
+
+from aidants_connect_web.models import Connection
+
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger()
+
+
+def fc_authorize(request):
+    fc_base = settings.FC_AS_FS_BASE_URL
+    fc_id = settings.FC_AS_FS_ID
+    fc_callback_uri = f"{settings.FC_AS_FS_CALLBACK_URL}/callback"
+
+    state = token_urlsafe(16)
+    connexion = Connection(state=state, connection_type="FS")
+    connexion.save()
+
+    fc_nonce = "customNonce11"
+    # TODO Make the nounce dynamic
+
+    fc_scopes = [
+        "given_name",
+        "family_name",
+        "preferred_username",
+        "birthdate",
+        "gender",
+        "birthplace",
+        "birthcountry",
+    ]
+
+    parameters = (
+        f"response_type=code"
+        f"&client_id={fc_id}"
+        f"&redirect_uri={fc_callback_uri}"
+        f"&scope={'openid' + ''.join(['%20' + scope for scope in fc_scopes])}"
```

Comment:
Why do you add an empty string here?

Reply:
Are you referencing `''.join`? 
- if yes : the expression:
```py
string = ''.join(['%20' + scope for scope in fc_scopes]
```
is equivalent to 
```py
string = ''
for scope in fc_scopes:
string = '' + '%20' + scope
```
the empty string is the separator as we don't want spaces in our url.
https://docs.python.org/2/library/string.html#string.join

however, if might work to write 
```suggestion
        f"&scope={'openid' + ''%20''.join([scope for scope in fc_scopes])}"
```

- If not: Could you tell me which empty string

Reply:
Yes that was the string, now I understand that it's used to initialize the string, thanks :)

Reply:
I made a mistake in the first version of my comment. I corrected it.
The empty string is not only initializing the string, it is also use to separate the different element of the list once it is in string form.

For example : 
```py
"".join([a, b, c])
# abc

", "..join([a, b, c])
# a, b, c

"%20".join([a, b, c])
# a%20b%20c
```